source/versionInfo.py

```diff
@@ -29,9 +29,9 @@
 URL: {url}
 {copyright}
 
-{name} is covered by the GNU General Public License (Version 2). You are free to share or change this software in any way you like as long as it is accompanied by the license and you make all source code available to anyone who wants it. This applies to both original and modified copies of this software, plus any derivative works.
+{name} is covered by the GNU General Public License (Version 2 or later). You are free to share or change this software in any way you like as long as it is accompanied by the license and you make all source code available to anyone who wants it. This applies to both original and modified copies of this software, plus any derivative works.
```

Comment:
Improve linting while we are modifying this

```suggestion
{name} is covered by the GNU General Public License (Version 2 or later).
You are free to share or change this software in any way you like as long as it is accompanied by the license and you make all source code available to anyone who wants it.
This applies to both original and modified copies of this software, plus any derivative works.
```